src/components/shared/FormHelpers.tsx

```diff
@@ -12,6 +12,11 @@ export function getTotalPercentage(mods: PayoutMod[] | undefined) {
   )
 }
 
+export function validateDistributionPercent(percent: number | undefined) {
```

Comment:
good move pulling this out as a shared validator. what do you think about giving this a more generic name since we could use it elsewhere, like `validateGreaterThanZero`?

Reply:
Yep I like it